src/v2/styles/AttachmentList/AttachmentList-layout.scss

```diff
@@ -1,34 +1,56 @@
 @use '../utils';
 
+.str-chat {
+  --str-chat__attachment-margin: var(--str-chat__spacing-0_5);
```

Comment:
Is this a "private" variable? If not, a brief comment about it would be helpful, as I am not sure where that margin would be applied.

Reply:
Done